### DISCUSSION TITLE
fill the void of empty search results

### DISCUSSION THREAD

**Discussion by neuromusic** [chanzuckerberg/napari-hub#246] [Ideas]
If I search for something that doesn't (yet) have any napari plugins, I'm left with an empty void...

![image](https://user-images.githubusercontent.com/1245615/130288068-d0642c9d-0fa7-4efd-b20b-389417e50262.png)

As we add categories, we'll probably see the same challenge in these early days (empty categories). We should consider filling this void with something useful for plugin users and/or plugin developers.

Ideas:
- Offer an alternative search or category that is similar but has more results
- Give biologists a way to "pitch a plugin"... e.g. "Can't find what you're looking for? Click here to propose a plugin."
- Prompt bioimage analysts to create a plugin... e.g. "It doesn't look like we have a plugin that does what you're looking for. Maybe you want to create one? Click here to get started."

**Comment by liaprins-czi** [comment]
Yeah I have thought of what we could have on the blank results area, and I really like your suggestion to "pitch a plugin"!

> we'll probably see the same challenge in these early days (empty categories)

I actually think we might want to only show categories with plugin(s) and hide potential categories (that devs could choose) from end users if they are empty. But there still might be empty results if user cross-filters multiple dimensions and if we have an "AND" operator between them

**Reply by neuromusic** [reply]
that's a good point!
